sdk/attestation/azure-security-attestation/inc/azure/attestation/attestation_client.hpp

```diff
@@ -225,24 +225,21 @@ namespace Azure { namespace Security { namespace Attestation {
     /**
      * @brief Perform a single leg
```

Comment:
This brief description (which is the highlighted opening paragraph) is not a complete sentence.